### PR TITLE
Ephemeral action

### PR DESCRIPTION
**Pull request by anr-tonic**
Example usage:
https://github.com/TonicAI/allos/blob/ac54eefe9ebc6a53f6d2a22d9d832aae01f9835b/.github/workflows/create-pr-environment.yaml#L77

A run:
https://github.com/TonicAI/allos/actions/runs/9197593845/job/25299246424

### CODE REVIEW

File: ephemeral/database/index.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  }
+
+  return {
+    databaseEntityId: database.databaseEntityId,
+    status: database.status,
+    host: `${database.kubernetesServiceName}.${database.kubernetesNamespace}`,
+    port: database.port,
+    databaseName: database.databaseName,
+    user: database.databaseUserName,
+    password: database.databasePassword,
```
Comment (anr-tonic) [inline ephemeral/database/index.js:19]: It looks like this is encrypted, potentially with a server wide key -- is there a way to reverse this?

File: ephemeral/database/setup.js
Comment (anr-tonic) [inline ephemeral/database/setup.js:1]: Ignore this; I didn't want `node_modules/` distracting from the actual action; it'll be replaced before merge.

File: ephemeral/database/index.js (hunk 1 above)
Comment (shaunmroberts) [reply ephemeral/database/index.js:19]: this should be the raw password, same response property we give to a user in the UI to be able to auth

**Review by shaunmroberts — APPROVED** [review]

File: ephemeral/database/index.js (hunk 1 above)
Comment (anr-tonic) [reply ephemeral/database/index.js:19]: I think I've misunderstood something in that case. 👍

**Pull request merged**